src/ATen/CMakeLists.txt

```diff
@@ -19,3 +19,7 @@ set(ATen_XPU_SYCL_SRCS ${ATen_XPU_SYCL_SRCS} PARENT_SCOPE)
 foreach(HEADER  ${xpu_h})
   install(FILES ${HEADER} DESTINATION "${AT_INSTALL_INCLUDE_DIR}/ATen/xpu")
 endforeach()
+
+foreach(HEADER  ${xpu_ops_generated_headers})
```

Comment:
@dvrogozh I have updated the missing headers here.

Reply:
Thank you, this seems to work now - I see headers installed.